doc/reference/modules/configuration.xml

```diff
@@ -1219,65 +1228,91 @@ in the parameter binding.</programlisting>
                         <row>
                             <entry>Microsoft SQL Server 7</entry>
                             <entry><literal>NHibernate.Dialect.MsSql7Dialect</literal></entry>
-                            <entry></entry>
+                            <entry>Use same default driver than <literal>NHibernate.Dialect.MsSql2000Dialect</literal>.</entry>
```

Comment:
I'm not sure what you mean here.  Is "article" an xml element for documents?  Or did you mean you should use a different word instead of "than"?  This would read better:

"Use the same default driver **as** `<literal>NHibernate.Dialect.MsSql2000Dialect</literal>`."

Reply:
The current sentence is `Use same default driver`, without `the` (which is what I have called an "article", in French "déterminant"). I think it should be `Use the same default driver`. About `than`, yes, better put `as` instead.

Reply:
Updated text and also added links to where the NuGet packages would be on nuget.org.